include/openssl/ct.h

```diff
@@ -481,7 +485,7 @@ int i2o_SCT_signature(const SCT *sct, unsigned char **out);
 * |len| should be the length of the signature in |in|.
 * Returns the number of bytes parsed, or a negative integer if an error occurs.
 */
-int o2i_SCT_signature(SCT *sct, const unsigned char **in, size_t len);
+__owur int o2i_SCT_signature(SCT *sct, const unsigned char **in, size_t len);
```

Comment:
On an unrelated note, at a glance I don't see any bindings between the signature/hash algorithm NID and the verification code (which could reject early if there's a mismatch, rather than attempt to verify an ECDSA signature with an RSA key). But this can be left for later polishing.